frontend/app_flowy/lib/plugins/grid/presentation/widgets/cell/date_cell/date_cell.dart

```diff
@@ -71,19 +72,30 @@ class _DateCellState extends GridCellState<GridDateCell> {
             margin: EdgeInsets.zero,
             child: SizedBox.expand(
               child: GestureDetector(
-                behavior: HitTestBehavior.opaque,
-                onTap: () => _popover.show(),
```

Comment:
Hello @Eakam1007, thanks for contributing! Your fix works well, but the `GridCellWidget` class already  provides this functionality. It just hasn't been hooked up yet. It should be enough to simply add a line to set `widget.onCellEditing.value` to `true` in this `onTap` handler.

Reply:
Oh, I missed that. I will update it, thanks

Reply:
I tried updating it and it works for a new cell. However, if you change the type of the cell to something else (such as text) and then back, the border stops showing.
This also happens for cells of type `Select` and `URL` where it is implemented in a similar way. Is this a known bug? I could not find an issue for this

Reply:
Yeah I can also reproduce this. Could you open a separate issue for that?

Reply:
Sure, I will update this branch and open a separate issue for this